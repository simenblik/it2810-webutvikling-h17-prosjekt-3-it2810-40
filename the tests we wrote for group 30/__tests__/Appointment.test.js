import React from 'react';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';
import Appointment from '../src/Components/Appointment.js';

/* Takes a snapshot of the rendered component and see if it
matches the original snapshot.
*/
describe('<Appointment/>, functions', () => {
	it('should match to previous snapshots', () => {
		const component = shallow(<Appointment />);
		const tree = toJson(component);
		expect(tree).toMatchSnapshot();
	});
});
