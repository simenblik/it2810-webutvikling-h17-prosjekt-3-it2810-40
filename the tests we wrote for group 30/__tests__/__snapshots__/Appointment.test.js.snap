// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Appointment/>, functions should match to previous snapshots 1`] = `
<div>
  <h1>
    Appointments
  </h1>
  <p>
    Click on an appointment to remove it!
  </p>
  <FormAppointment />
</div>
`;
